Movies_MVVMTests/ImageNetworkServiceTests.swift

```diff
@@ -0,0 +1,58 @@
+// ImageNetworkServiceTests.swift
+// Copyright © Aleksandr Dorofeev. All rights reserved.
+
+import XCTest
+
+@testable import Movies_MVVM
+
+/// Image network service tests
+final class ImageNetworkServiceTests: XCTestCase {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let emptyString = ""
+    }
+
+    // MARK: - Private properties
+
+    private var imageNetworkService: ImageNetworkServiceProtocol?
+
+    // MARK: - Public methods
+
+    override func setUp() {
+        imageNetworkService = MockImageNetworkService()
+    }
+
+    override func tearDown() {
+        imageNetworkService = nil
+    }
+
+    func testSuccessImageRequest() {
+        let mockImageData = Data()
```

Comment:
```suggestion
        let mockImageData = Data()
```
Data() - 0 byte, Data(count: 1) - 1 byte/ Это не замечание

Reply:
Мне важно было просто проинициализировать дату, что бы проверить заходит ли метод в условие, а сколько байт я передаю без разницы, главное чтобы не был нил